RELEASE.md

```diff
@@ -1,31 +1,31 @@
-# RELEASE NOTES for FV3: Summary
+# RELEASE NOTES for GFDL  FV3: Summary
 
-FV3-201912-public --- 10 January 2020
-Lucas Harris, GFDL <lucas.harris@noaa.gov>
+2020.02 --- 22 April 2020
 
-This version has been tested against the current GFDL AM4 phsyics
-and with release candidate 2020.02 of the FMS code at https://github.com/NOAA-GFDL/FMS
+This version has been tested within current GFDL Models (AM, CM, ESM, SPEAR, etc.) and requires the 2020.02 release of the [FMS infrastructure](https://github.com/NOAA-GFDL/FMS).
 
-Includes all of the features of the GFDL Release to EMC, as well as:
+Includes all of the features of the [201912 Public Release](https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/releases/tag/201912_public_release) which include:
```

Comment:
I think the tag is wrong.  Shouldn't it be 2020.02?

Reply:
@thomas-robinson The tag is correct.  The 2020.02 release includes the new nesting  merged in from the 201912_public_release tag on the master branch.